__tests__/operation/get-parameters-as-json-schema.test.ts

```diff
@@ -430,35 +434,6 @@ describe('required', () => {
   it.todo('should make things required correctly for request bodies');
 });
 
-describe('`globalDefaults` option', () => {
```

Comment:
Moved this to the bottom of the file and put it in a new `describe` block for all of the options that are now available.